### ISSUE TITLE
Users need to print and keep the information on completion page

### ISSUE THREAD

**Issue by EUzkuraityte** [alphagov/verify-local-patterns#595]
UR evidence from Cambridgeshire
link to #470 
![screen shot 2017-06-16 at 16 34 56](https://user-images.githubusercontent.com/20285115/27233562-c3051490-52b1-11e7-9345-e522caa92325.png)

**Comment by lizziebruce** [comment]
Is this a real need? Most users have said they would take a screenshot or print the page themselves. The email notification says confirmation coming in the next minute or prompts user to sign up for email notifications if they haven't.

If it is a real need, suggest we add a save as PDF / print as PDF. But wouldn't most users know how to do that?

**Comment by EUzkuraityte** [comment]
Hi @lizziebruce yup, definitely sounds good! Sanjay just [showed me a page](https://www.gov.uk/social-security-child-support-tribunal) that has similar functionality on GOV.UK, maybe we can make it similar to this?
![screen shot 2017-06-26 at 13 44 59](https://user-images.githubusercontent.com/20285115/27539818-db64f0b6-5a75-11e7-9765-3eb1523ca36f.png)

**Comment by lizziebruce** [comment]
Yeah exactly, but with the link saying 'Print this page' right? We'll just have to make sure it's not confusable with the 'Print temporary permit' link...

**Comment by EUzkuraityte** [comment]
Tagging both @lizziebruce and @sanjaypoyzer as Lizzie advised on the proposed copy in the previous comment and it's to be implemented by whoever can do it first..
